_layouts/post.html

```diff
@@ -32,6 +40,13 @@ <h1 class="post-title">{{ page.title }}</h1>
                 {{ content }}
 
                 {% include pagination-post.html %}
+                <div class="tags" id="share-with">
```

Comment:
share with should be for shared part right?
why we are moving to tags?

Reply:
after navigation, it is not showing share heading it is not good so moved it to tags

Reply:
oh cool.
merge this.